cmd/aws/main.go

```diff
@@ -230,3 +236,32 @@ func storageConfigFromFlags() taws.Config {
 		MaxIdleConns: *dbMaxIdle,
 	}
 }
+
+func antispamMysqlConfig() *mysql.Config {
+	if *antispamDBName == "" {
+		klog.Exit("--antispam_db_name must be set")
+	}
+	if *dbHost == "" {
+		klog.Exit("--db_host must be set")
+	}
+	if *dbPort == 0 {
+		klog.Exit("--db_port must be set")
+	}
```

Comment:
The default `dbPort` is 3306 instead of 0.

Reply:
Yes, this is working as intended: it's ok to use the default port, but 0 would indicate that it's the default integer.

Reply:
If `--db_port` is not set, the `*dbPort` value would be `3306`. The log message is not valid.